doc/guide.rst

```diff
@@ -305,6 +305,15 @@ Special arguments
   A use case for this is wrapping of scripts that are completely ``Argbash``-agnostic.
   Therefore, your script can take its own arguments and the rest that is not recognized can go to the wrapped script.
 
+* Detect supplied arguments:
+  ::
+
+    ARGBASH_INDICATE_SUPPLIED([long arg name], ...)
+
+  This macro will cause a variable to be set if the argument was explicitly provided. This only works for optional arguments.
```

Comment:
Please also indicate that the macro takes at least one argument, and subsequent values are treated in the same way as subsequent calls to the macro with one argument. Or something like that, if you can phrase it better.